__tests__/app.test.js

```diff
@@ -93,33 +93,88 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/not-an-id")
       .expect(400)
       .then((res) => {
-        console.log(res.body.msg, "<body");
         expect(res.body.msg).toBe("Invalid input");
       });
   });
 });
 
-  describe("GET /api/articles", () => {
-    it("return array of articles with following properties", () => {
+describe("GET /api/articles", () => {
+  it("return array of articles with following properties", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then((response) => {
+        const {
+          body: { articles },
+        } = response;
+        expect(Array.isArray(articles)).toBe(true);
+        expect(articles).toBeSorted({ descending: true });
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("author");
+          expect(article).toHaveProperty("title");
+          expect(article).toHaveProperty("article_id");
+          expect(article).toHaveProperty("topic");
+          expect(article).toHaveProperty("created_at");
+          expect(article).toHaveProperty("votes");
+          expect(article).toHaveProperty("article_img_url");
+          expect(article).toHaveProperty("comment_count");
```

Comment:
There is a nicer matcher to use here. Check out the jest docs when you get chance.